app/services/decisions/dsl_decision_tree.rb

```diff
@@ -2,6 +2,17 @@ module Decisions
   class DslDecisionTree < BaseDecisionTree
     WRAPPER_CLASS = SimpleDelegator
 
+    NSM_FIRM_DETAILS = 'nsm/steps/firm_details'.freeze
+    NSM_WORK_ITEMS = 'nsm/steps/work_items'.freeze
+    NSM_WORK_ITEM = 'nsm/steps/work_item'.freeze
+    NSM_DEFENDANT_SUMMARY = 'nsm/steps/defendant_summary'.freeze
+    NSM_DISBURSEMENTS = 'nsm/steps/disbursements'.freeze
+    NSM_DISBURSEMENT_TYPE = 'nsm/steps/disbursement_type'.freeze
+    NSM_CLAIM_DETAILS = 'nsm/steps/claim_details'.freeze
+    NSM_LETTERS_CALLS = 'nsm/steps/letters_calls'.freeze
+    NSM_DISBURSEMENT_ADD = 'nsm/steps/disbursement_add'.freeze
+    NSM_EQUALITY = 'nsm/steps/equality'.freeze
```

Comment:
I would be expect these at the top of `app/services/decisions/decision_tree.rb` (with the plan to move this class into the gem if we ever get to it being resused.. :) 

as feel this is more generic base code

Reply:
That's a great point. I was thinking that these constants are used in `BackDecisionTree` as well as `DecisionTree`, but you're right, `DslDecisionTree` is generic, and I think it's OK for BackDecisionTree to reference constants defined in DecisionTree (because hopefully we'll be able to eliminate BackDecisionTree at some point).